src/packages/core/collection/default/collection-default.context.ts

```diff
@@ -120,11 +120,14 @@ export class UmbDefaultCollectionContext<
 			this.pagination.setPageSize(this.#config.pageSize);
 		}
 
+		const filterValue = this.#filter.getValue() as FilterModelType;
+
 		this.#filter.setValue({
 			...this.#defaultFilter,
 			...this.#config,
 			...this.#filter.getValue(),
```

Comment:
It looks like we append this value twice. 🤔

Reply:
Doh! 🤦  Thanks @madsrasmussen, I was checking if anybody noticed. 👀  I'll fix it up.